app/assets/stylesheets/thredded/components/_post.scss

```diff
@@ -78,7 +78,7 @@
     @extend %thredded--blockquote;
   }
 
-  .spoiler {
+  .thredded--post--content--spoiler {
```

Comment:
hahaha oh jeeeez how the hell did I miss that

On Mon, May 9, 2016 at 12:56 PM, Gleb Mazovetskiy <notifications@github.com>
wrote:

> In app/assets/stylesheets/thredded/components/_post.scss
> <https://github.com/thredded/thredded/pull/287#discussion_r62533102>:
>
> > @@ -78,7 +78,7 @@
> >      @extend %thredded--blockquote;
> >    }
> >
> > -  .spoiler {
> > +  .thredded--post--content--spoiler {
>
> Since this is a fully qualified class, it no longer needs to be nested,
> so: &--spoiler
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/thredded/thredded/pull/287/files/7a45ed117d1fab28446295079dd5216cace8a688#r62533102>
>